### PR TITLE
add fcm base implementation

### PR DESCRIPTION
**Pull request by RyuNen344**
## Issue
-  #41

## Overview (Required)
- I made base implementation
- I will make repository and api after this PR

[Q]
are there any more information of notification channel id and payload specific...?

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

**Comment by takahirom** [comment]
Thank you for the step-by-step PR! Small PR is great!!

File: android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package io.github.droidkaigi.feeder.notification
+
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class KaigiMessagingService : FirebaseMessagingService() {
```
Comment (takahirom) [inline android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt:8]: I haven't used Kaigi ~ so far, so I'd like to change the name to AppMessagingService later, but you don't need to change it with this PR. 👀

File: android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package io.github.droidkaigi.feeder.notification
+
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
```
Comment (takahirom) [inline android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt:7]: 👍

**Comment by takahirom** [comment]
> are there any more information of notification channel id and payload specific...?

Let's implement it once with this PR implementation and fix it later! I haven't decided much about this.

File: android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class KaigiMessagingService : FirebaseMessagingService() {
+
+    override fun onCreate() {
+        NotificationUtil.createNotificationChannel(this)
+    }
+
+    override fun onNewToken(token: String) {
+        // TODO: 2021/02/28 implement sending server
```
Comment (takahirom) [inline android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt:15]: Will you implement this? If not, can you create an issue for this? 🙏

**Review by takahirom — APPROVED** [review]
Thank you for the minimum PR for improving!

File: android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt (hunk 3 above)
Comment (RyuNen344) [reply android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt:15]: yes! ok, i will diassemble and make pr

**Pull request merged**